.github/workflows/docs-language-linter.yaml

```
@@ -0,0 +1,33 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: Docs Language Linter
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+      - '**.md'
+      - ".github/workflows/docs-language-linter.yaml"
+      - ".vale.ini"
+      - ".github/styles/**"
+
+jobs:
+  vale:
+    name: Review docs with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: errata-ai/vale-action@reviewdog
```

Is there a specific reason for taking their latest commit and not an official release?
```suggestion
      - uses: errata-ai/vale-action@v2
```

---

@jond01 no special reason actually, I can change to `v2`

---

A version can be more stable, but since they give this example it's ok. So whatever you think is better :)

---

@jond01 let's leave it as is now, I see that they don't merge changes too often :)